### PR TITLE
Update graphics for L5.10.72-2.2.0

### PR DESCRIPTION
**Pull request by thochstein**

### CODE REVIEW

File: recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc [hunk 1]
```diff
@@ -190,16 +189,8 @@ do_install () {
         install -m 0644 ${S}/gpu-core/usr/lib/pkgconfig/glesv2.pc ${D}${libdir}/pkgconfig/glesv2.pc
         install -m 0644 ${S}/gpu-core/usr/lib/pkgconfig/vg.pc ${D}${libdir}/pkgconfig/vg.pc
 
-       if [ "${USE_X11}" = "yes" ]; then
```
Comment (otavio) [inline recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc:193]: So X11 is dead? please add it to commit log.

File: recipes-graphics/wayland/weston-init.bbappend [hunk 2]
```diff
@@ -49,3 +49,15 @@ do_install:append() {
...
+WATCHDOG_SEC_mx8ulp = "240"
+replace_setting() {
+    if ! grep -q "$1" $3; then
+        bbwarn "Setting to replace '$1' not found in file $3"
+    fi
+    sed -i -e "s,$1,$2," $3
+}
+do_install:append() {
+    replace_setting "WatchdogSec=.*" "WatchdogSec=${WATCHDOG_SEC}" ${D}${systemd_system_unitdir}/weston.service
+}
```
Comment (otavio) [inline recipes-graphics/wayland/weston-init.bbappend:63]: This isn't a BSP thing; please move it to meta-freescale-distro and also restrict it for fsl distros.

**Review by otavio — CHANGES_REQUESTED** [review]

**Comment by thochstein** [comment]
Ping.

File: recipes-graphics/wayland/weston-init.bbappend (hunk 2 above)
Comment (thochstein) [reply recipes-graphics/wayland/weston-init.bbappend]: Done: https://github.com/Freescale/meta-freescale-distro/pull/80

**Review by otavio — APPROVED** [review]

**Pull request merged**